config/locales/en.yml

```diff
@@ -829,72 +829,72 @@ en:
+          apartments: "Apartment Building"
           barn: "Barn"
           bungalow: "Bungalow"
           cabin: "Cabin"
-          chapel: "Chapel"
+          chapel: "Chapel Building"
           church: "Church Building"
           civic: "Civic Building"
           college: "College Building"
           commercial: "Commercial Building"
-          construction: "Building under Construction"
+          construction: "Building Under Construction"
           cowshed: "Cowshed"
           detached: "Detached House"
           dormitory: "Dormitory"
           duplex: "Duplex House"
-          farm: "Farm House"
-          farm_auxiliary: "Auxiliary Farm House"
+          farm: "Farmhouse"
+          farm_auxiliary: "Farm Building"
           garage: "Garage"
           garages: "Garages"
           greenhouse: "Greenhouse"
-          hangar: "Hangar"
+          hangar: "Hangar Building"
           hospital: "Hospital Building"
           hotel: "Hotel Building"
           house: "House"
           houseboat: "Houseboat"
           hut: "Hut"
           industrial: "Industrial Building"
           kindergarten: "Kindergarten Building"
-          manufacture: "Manufacturing Building"
+          manufacture: "Industrial Production Building"
           office: "Office Building"
           public: "Public Building"
           residential: "Residential Building"
           retail: "Retail Building"
           roof: "Roof"
-          ruins: "Ruined Building"
+          ruins: "Building Ruins"
           school: "School Building"
-          semidetached_house: "Semidetached House"
+          semidetached_house: "Semi-Detached House"
           service: "Service Building"
           shed: "Shed"
           stable: "Stable"
-          static_caravan: "Caravan"
-          sty: "Sty"
+          static_caravan: "Mobile Home"
+          sty: "Pigsty"
           temple: "Temple Building"
-          terrace: "Terrace Building"
+          terrace: "Rowhouses"
           train_station: "Train Station Building"
           university: "University Building"
           warehouse: "Warehouse"
           "yes": "Building"
         club:
-          scout: "Scout Group Base"
+          scout: "Scout Group"
           sport: "Sports Club"
           "yes": "Club"
         craft:
           beekeeper: "Beekeeper"
           blacksmith: "Blacksmith"
-          brewery: "Brewery"
+          brewery: "Craft Brewery"
           carpenter: "Carpenter"
           caterer: "Caterer"
-          confectionery: "Confectionary"
+          confectionery: "Candymaker"
           dressmaker: "Dressmaker"
           electrician: "Electrician"
-          electronics_repair: "Electronics Repair"
+          electronics_repair: "Electronics Repair Shop"
           gardener: "Gardener"
           glaziery: "Glaziery"
-          handicraft: "Handicraft"
-          hvac: "HVAC Craft"
-          metal_construction: "Metal Constructioner"
+          handicraft: "Handicraft Workspace"
```

Comment:
[This Wikipedia page](https://en.wikipedia.org/wiki/Handicraft) offers some alternative names. `Handcrafting Workspace` or `Artisan` would sound more natural to my ear.